Vagrantfile

```diff
@@ -32,8 +32,7 @@ Vagrant.configure(2) do |config|
         wget -qO- $ANDROID_SDK_URL | tar xz -C tools
         expect -c '
             set timeout -1;
-            spawn ./tools/android-sdk-linux/tools/android update sdk --no-ui \
-                -t tool,platform-tool,platform,build-tools-24.0.1;
```

Comment:
When I added `build-tools-24.0.1` it was the current version.  Newer Android SDK include build-tools-24.0.2 and they fail to install 24.0.1 :flushed: